package.json

```diff
@@ -39,6 +39,7 @@
 		"@types/node": "^20.10.4",
 		"@vercel/ncc": "^0.38.1",
 		"prettier": "^3.1.0",
+		"mock-fs": "^5.4.0",
```

Comment:
Used by tests to mock the filesystem, but open to another approach for this that doesn't involve adding a new package!

Reply:
I thought this approach was a really simple and readable one though

Reply:
This is awesome! Since it's just used in tests, we can probably make it a `devDependencies`?

Reply:
I think it is, right?

Reply:
Oops! I was not paying attention!!